examples/modules/thermoelectric/pages/electromagnet/README.adoc

```diff
@@ -0,0 +1,83 @@
+= Description of the example
+:page-vtkjs: true
+
+In this example, we will estimate the rise in temperature due to Joules losses in a stranded conductor. An electrical potential \$V_D\$ is applied to the entry/exit of the conductor which is also cooled by a force flow.
+
+== Geometry
+
+- The conductor consists in a solenoid, which is one helix of a magnet.
+- image:electromagnet/electromagnet-geo.png[geo.png,height=283]
+- Girder ID: 5af59e88b0e9574027047fc0
+
```

Comment:
Perhaps state the obvious that the main fields of concern wil be T and V at some point